lib/exception_notification/rack.rb

```diff
@@ -23,6 +23,13 @@ def initialize(app, options = {})
         end
       end
 
+      if options.key?(:ignore_notifier_if)
```

Comment:
Had to set a new option key that's different from `ignore_if`, as current `ignore_if` needs to accept two-arguments lambda, while the new API requires three.